nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-nar-utils/src/main/java/org/apache/nifi/nar/StandardExtensionDiscoveringManager.java

```diff
@@ -128,7 +131,11 @@ public void discoverExtensions(final Set<Bundle> narBundles) {
             // so that static initialization techniques that depend on the context class loader will work properly
             final ClassLoader ncl = bundle.getClassLoader();
             Thread.currentThread().setContextClassLoader(ncl);
+
+            final long loadStart = System.currentTimeMillis();
             loadExtensions(bundle);
+            final long loadMillis = System.currentTimeMillis() - loadStart;
+            logger.info("Loaded extensions for {} in {} millis", bundle.getBundleDetails(), loadMillis);
```

Comment:
Should this timing information be logged at the info level, that can produce a number of logs given that it runs for each bundle.

Reply:
I did consider moving that down to DEBUG level. But I think it makes more sense at INFO level. It is verbose, yes. But it's only on startup, and can be good info to have, as a long load time would indicate that something may be problematic. Plus, it was certainly convenient for measuring timing before & after the change. But I am ok decreasing to DEBUG if you think it would be annoying in the logs.

Reply:
Thanks for the explanation, I'm fine with leaving it at the INFO level since it is only at startup, just wanted to make sure it was intentional.